arcgis-ios-sdk-samples/Maps/Map reference scale/README.metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+    "category": "Maps",
+    "description": "@@TODO: Set the map's reference scale and which feature layers should honor the reference scale.",
+    "ignore": false,
+    "images": [
+        "image1.png",
+        "image2.png"
+    ],
+    "keywords": [
+        "Maps",
+        "and",
```

Comment:
Instead of converting "&" to "and", it might be better to just omit it as a keyword altogether.

Reply:
oh, good catch - I forget to add back a processing step which I took out for the android metadata files.